packages/web-app/src/App.jsx

```diff
@@ -90,7 +92,27 @@ const App = () => (
               <Provider store={gcStore}>
                 <HydratedIntlProvider onError={customOnIntlError}>
                   <ErrorHandler />
-                  <ErrorBoundary>
+                  <ErrorBoundary
+                    fallback={
+                      <Box
+                        width="100vw"
+                        height="100vh"
+                        display="flex"
+                        justifyContent="center"
+                        alignItems="center"
+                        flexDirection="column">
+                        <Typography variant="h1" color="error">
+                          <Translate>
+                            Sorry, something went really wrong...
+                          </Translate>
+                        </Typography>
+                        <Typography>
+                          <Translate>
+                            Refresh the page to browse Grottocenter again.
```

Comment:
Ne faudrait il pas plutot écrire : 
`<a>Go back to the home page of Grottocenter</a>`
car j'ai peur que le refresh fasse revenir la meme erreur car on sera surement sur l'URL qui sera problématique non ?

Reply:
On pourrait également proposer un lien <a>Go back to the previous page you visited</a>